frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -101,14 +102,16 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
         </p>
         <Row>
           <Col className="marker-svg">
-            <SearchMarkerButton
-              onClick={(e: any) => {
-                setMovingPinNameSpace(nameSpace ?? '');
-                e.preventDefault();
-              }}
-            >
-              <ParcelDraftIcon className="parcel-icon" />
-            </SearchMarkerButton>
+            <ClickAwayListener onClickAway={() => setMovingPinNameSpace(undefined)}>
```

Comment:
This is a more cohesive way of ensuring that the cursor gets reset when anything on the screen is clicked.